projects/systelab-components/src/lib/spinner/spinner.component.spec.ts

```diff
@@ -14,15 +14,16 @@ import { TouchspinComponent } from './spinner.component';
 	selector: 'systelab-spinner-test',
 	template: `
                 <div>
-                    <systelab-spinner [spinValues]="values" (change)="doValueChange()"></systelab-spinner>
+                    <systelab-spinner [spinValues]="values" (change)="doValueChange()" [fillUnitsWithZero]="fillUnitsWithZero" [(valueStr)]="strValue" ></systelab-spinner>
                     <label class="label-value">{{values.value}}</label>
                 </div>
 	          `,
 	styles:   []
 })
 export class SpinnerTestComponent {
 	public values = new TouchSpinValues(34, 1, 100);
-	
+	public fillUnitsWithZero: boolean | number =false;
+	public strValue: string;
```

Comment:
Use the same variable name "valueStr" for consistency.

Reply:
done